### DISCUSSION TITLE
More than one HA Instance

### DISCUSSION THREAD

**Discussion by Derity1** [homeassistant-apps/app-cloudflared#722] [Ideas]
I would like to have a domain example.de with two subdomains, like home1.example.de and home2.example.de, and connect them to two independent Home Assistant instances.

home1.example.de was created successfully, but home2.example.de failed with this message: [20:06:08] FATAL: Failed to create tunnel. Please check the Cloudflare Zero Trust Dashboard for an existing tunnel with the name homeassistant and delete it:

To correct this, visit https://one.dash.cloudflare.com/, then click on Access / Tunnels.

Or is there a simple way to rename the second tunnel?

**Comment by Derity1** [comment]
sorry Found the Feature, Thanks

**Comment by brenner-tobias** [comment]
For future reference: Yes, use the configuration option [tunnel_name](https://github.com/brenner-tobias/addon-cloudflared/blob/main/cloudflared/DOCS.md#option-tunnel_name) or set-up a remote managed tunnel and rename them in the Cloudflare Zero Trust Dashboard
